### PR TITLE
added button disabling for auth and ran lint

### PR DESCRIPTION
**Pull request by finlaylp**
Nothing big, just added button disabling.

### CODE REVIEW

File: components/Authentication/login.jsx [hunk 1]
```diff
@@ -147,14 +154,27 @@ const Login = ({ navigation }) => {
...
+            uppercase={false}
+            mode="contained"
+            dark
+            onPress={loginPress}
+          >
+            <LoginText>Login</LoginText>
+          </LoginButton>
+        )
+        : (
+          <DisableLoginButton
```
Comment (ctperry0301) [inline components/Authentication/login.jsx:167]: Would u want this button to have a disabled={disableButton} like you do the sign up one?

**Review by ctperry0301 — COMMENTED** [review]
Im pretty sure there is a better way to conditionally render a style than to use a ternary for the whole button. The change works great and looks good tho.

File: components/Authentication/login.jsx (hunk 1 above)
Comment (finlaylp) [reply components/Authentication/login.jsx:167]: yeah good catch

**Comment by finlaylp** [comment]
> Im pretty sure there is a better way to conditionally render a style than to use a ternary for the whole button. The change works great and looks good tho.

Yeah there must be I just couldn't figure it out and this seemed like an easy fix anyway

**Review by ctperry0301 — APPROVED** [review]
Looks Good.

**Pull request merged**